tests/test_strict_claim_service_mux.py

```
@@ -0,0 +1,127 @@
+class _Claim:
+    async def research_case(self, case_context: str, language: str = "ru"):
+        return ("claim-research", case_context, language)
+
+    async def draft_claim(self, case_context: str, research, language: str = "ru"):
+        return ("claim-draft", case_context, research, language)
+
+
+def test_claim_service_mux_routes_only_claim_methods():
+    stable = _Stable()
+    claim = _Claim()
+    service = ClaimServiceMux(stable, object(), claim_factory=lambda settings: claim)
```

Ruff (0.16.1)

[warning] Unused lambda argument: `settings`

(ARG005)